psalm.xml

```diff
@@ -26,6 +27,27 @@
                 <referencedMethod name="Laminas\Cache\Storage\Adapter\RedisClusterOptions::setResourceManager"/>
             </errorLevel>
         </InternalMethod>
+        <!--
+            Since these are usually suppressed anyway AND psalm is already thinking about removing this issue at all,
+            suppressing these might be a good idea. See https://github.com/vimeo/psalm/pull/8776 for more information.
+        -->
+        <MixedAssignment>
+            <errorLevel type="suppress">
+                <directory name="*"/>
+            </errorLevel>
+        </MixedAssignment>
+        <!--
+            Mark these redundant casts as info as some docblocks are not accurate about the return values.
```

Comment:
Similar here: would be best to let the baseline shrink with time, rather than hiding details.

I agree that some types may be inaccurate, but I'd expect shrinkage as the declarations get refined in upstream